vmware_aria_operations_integration_sdk/adapter_template/blank_template_adapter.py

```diff
@@ -0,0 +1,169 @@
+#  Copyright 2022 VMware, Inc.
+#  SPDX-License-Identifier: Apache-2.0
+import sys
+from typing import List
+
+import aria.ops.adapter_logging as logging
+from aria.ops.adapter_instance import AdapterInstance
+from aria.ops.definition.adapter_definition import AdapterDefinition
+from aria.ops.result import CollectResult
+from aria.ops.result import EndpointResult
+from aria.ops.result import TestResult
+from aria.ops.timer import Timer
+from constants import ADAPTER_KIND
+from constants import ADAPTER_NAME
+
+logger = logging.getLogger(__name__)
+
+
+def get_adapter_definition() -> AdapterDefinition:
+    """
+    The adapter definition defines the object types and attribute types (metric/property) that are present
+    in a collection. Setting these object types and attribute types helps VMware Aria Operations to
+    validate, process, and display the data correctly.
+    :return: AdapterDefinition
+    """
+    with Timer(logger, "Get Adapter Definition"):
+        definition = AdapterDefinition(ADAPTER_KIND, ADAPTER_NAME)
+
+        # The key 'container_memory_limit' is a special key read by the VMware Aria Operations
+        # collector to determine how much memory to allocate to the docker container running this
+        # adapter. It does not need to be read inside the adapter code. If the adapter doesn't
+        # read the key, the container running this adapter will be given 1024 MB of memory.
```

Comment:
The adapter never needs to read the key. It's Aria Ops that reads the key. (By the time the adapter is running, the container is already created, since the adapter is running inside it)

Reply:
Yes, that is correct